.travis.yml

```diff
@@ -14,6 +14,7 @@ env:
   global:
     # The BINSTAR_TOKEN secure variable. This is defined canonically in conda-forge.yml.
     - secure: "Rsc1XQLxRBgjpg44G1LYdAHpnYMs3K4KA0k5vU+bd6xEwaPx+Qj3AZFaf2wXyaB/pLn1BGkYCUbnoEoCiGTMhGaiiuf1G/eSUbI4jLUryDhJL08kLl83dPjOJE5gZ/nw/ZPbgEkAKWLM/+Y6ZRIqDsxZKSPxfw72OohsIKcWk5ASpdFfUecuFQw/Da2By5DSgWKsUyZhA848nx3vEn0pNktmRjH4L/cFWmphlDKnXifMaKi4HjSHA1yY3/JLVYkKOeNxEBNUYvcrUvtLXqdjfXZQnhMer16xgigqY3LRxFb91UCTBAD9oRIvZgG5N+cqAuTPwyUDRb6wTX6LcEcIOtD7XyVtiZpNTiE/LhH1yqJPBqgUNqUcWV9PDpmsFmeUsbnLYRS/4ohdJeA/LGghWIRzg2s99bZWsi4+NeTQSlrPfZFNW8fuNOLfCFmG+MVc9nQWNFeBhix1KVzy5SAqyxcr/k7QwnZQ9ov+hzNvwW74gkTQCWsG8gHvdCOSHW7mozcCVjcD5tWZ8OQuZVQPC9KZUC08iEV0pLOKgeRX+fMWCvaLq9Q93S+xifQpMzBPUcPIU/8WuuRxJfGwnDnaCGkBJvqdDCCo4dG8aZnT9lBo4+XC2wWzjqZuk3Ckc+/PlCiq6A4ZogrAmPyRV3m8UtgOJ/dXmlK0SdxQxgT9bV8="
+    - MAKEFLAGS="-j 2"
```

Comment:
We already have `make -j${CPU_COUNT}` in build.sh. Does this do something different?